components/mailer.rst

```diff
@@ -142,6 +142,17 @@ The mailer will start using the first transport and if it fails, it will retry
 the same delivery with the next transports until one of them succeeds (or until
 all of them fail).
 
+Disable TLS peer verification
+-----------------------------
+
+Symfony's mailer supports TLS peer verification by default, but in some cases, 
```

Comment:
SMTP tranports supports TLS peer verification by default, but in some cases, you may want to turn this off. Ie in a development environment.